src/pages/AnalyticsPage/ChartSelectors/ChartSelectors.jsx

```diff
@@ -126,11 +130,17 @@ function ChartSelectors({ params, updateParams, active, sortItem }) {
         resetRef={resetRef}
       />
       <button
-        className="mt-7 text-ds-blue-darker"
+        className="self-end text-ds-blue-darker md:mr-auto"
         onClick={clearFiltersHandler}
       >
         Clear filters
       </button>
+      {tierData === TierNames.TEAM ? (
+        <p className="self-end">
+          Public repos only. <A to={{ pageName: 'upgradeOrgPlan' }}>Upgrade</A>{' '}
+          to Pro to include private repos
```

Comment:
Should we include a period at the end here since technically it is a sentence? What do you think?

Reply:
Done thanks for the check!